code/datums/world_topic.dm

```
@@ -84,8 +94,11 @@
 /datum/world_topic/comms_console
 	keyword = "Comms_Console"
 	require_comms_key = TRUE
+	permit_insecure = TRUE
 
 /datum/world_topic/comms_console/Run(list/input, addr)
+	if(CHAT_FILTER_CHECK(input["message"])) // prevents any.. diplomatic incidents
```

I kind of hate that this silently fails. Maybe do some fluff announcement instead that a message was intercepted by the Centcom Department of Truth or something?

---

lmao alright

---

Centcom department of social justice

---

@TheFakeElon 1984 reference is funnier

---

current year reference is funnier, I'm going with the CCC (CentCom Communications Commission)